cwl/workflow/data_assembly_RNA_wf.cwl

```diff
@@ -0,0 +1,76 @@
+cwlVersion: v1.0
+class: Workflow
+id: data_assembly_RNA_wf
+requirements:
+  - class: MultipleInputFeatureRequirement
+  - class: SubworkflowFeatureRequirement
+inputs:
+
+  # format fusion
+  FusionGenome: { type: File }
+  arriba_output_file: { type: File }
+  star_fusion_output_file: { type: File }
+  old_arriba: { type: File }
+  old_starfusion: { type: File }
+
+  # rsem 
+  input_rsem: { type: File }
+  old_rsem_count: { type: File }
+  old_rsem_fpkm: { type: File }
+  old_rsem_tpm: { type: File }
+  library:  {type: ['null', {type: enum, name: library, symbols: ["polya", "stranded"]}]}
+
+  biospecimen_id_RNA: string
+
+outputs:
+  new_merged_star_fusion: { type: File, outputSource: merge_fusion/output_merged_star_fusion }
+  new_merged_merged_arriba:  { type: File, outputSource: merge_fusion/output_merged_arriba }
+  
+  new_merged_rsem_count: { type: File, outputSource: merge_rsem/output_merged_rsem_count }
+  new_merged_fpkm: { type: File, outputSource: merge_rsem/output_merged_fpkm }
+  new_merged_tpm: {type: File, outputSource: merge_rsem/output_merged_tpm}
+
+steps:
+  format_fusion:
+    run: ../tools/format_fusion.cwl
+    in:
+      arriba_output_file: arriba_output_file
+      star_fusion_output_file: star_fusion_output_file
+      biospceimens_id: biospecimen_id_RNA
```

Comment:
```suggestion
      biospecimen_id: biospecimen_id_RNA
```
updated to match the input name:
https://github.com/d3b-center/d3b-bixu-data-assembly/blob/41ffd69f374fd8e6d4e1a8cb2921833cfe6877eb/cwl/tools/format_fusion.cwl#L21